pom.xml

```diff
@@ -125,6 +143,10 @@
             <artifactId>junit-jupiter-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
Specifying this duplicate dependency currently causes Maven warnings which should be resolved with an OSS parent update.

Reply:
Guava was already used in test classes and is specified in the OSS parent so I wasn't aware of this intention.

The current Guava usage should not be exposed in the public API, though.
What concerns do/did you have?

Reply:
I've removed the Guava dependency in non-test code.